### PR TITLE
Nesterov

### PR DESCRIPTION
**Pull request by nester256**
Add hw2

### CODE REVIEW

File: second/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        -e POSTGRES_USER=admin \
+        -e POSTGRES_PASSWORD=1234 \
+        -e PGDATA=/postgres_data \
+        -v ~/djangoproject_hw2/postgres_data:/postgres_data \
+        -p 38746:5432 \
+        postgres:15.1
+```
+### Third step:
+```
+./manage.py migrate
```
Comment (siriusdevs) [inline second/README.md]: 1) we only changed directory to "second" , by manage.py is not there. its in the "restaurants" directory
2) you suggest running manage.py as a script, but it won't run like this as shabang in the beginning says `#!/usr/bin/env python`, and manage.py wont run with python, it must me python3

**Review by siriusdevs — CHANGES_REQUESTED** [review]
even if I manage to run your project, it will fall with 
`  File "/home/sirius/.local/lib/python3.11/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: FATAL:  database "restaurants_test" does not exist`

because you do not name your DB when creating a docker container

File: second/README.md (hunk 1 above)
Comment (nester256) [reply second/README.md]: Sorry for stupid mistakes, already corrected

**Comment by siriusdevs** [comment]
> even if I manage to run your project, it will fall with ` File "/home/sirius/.local/lib/python3.11/site-packages/psycopg2/__init__.py", line 122, in connect conn = _connect(dsn, connection_factory=connection_factory, **kwasync) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ django.db.utils.OperationalError: FATAL: database "restaurants_test" does not exist`
> 
> because you do not name your DB when creating a docker container

still not fixed.
are you enjoying this?

**Pull request merged**